9-2-Coaching-Expectations.md

```diff
@@ -0,0 +1,24 @@
+We need coaches to hover around and help our participants with any questions or other issues they might have as they go through the course content. There are a few roles our coaches can take on throughout the day:
+
+# Coaching roles
+
+## Slack channel monitors
+
+Some people will feel comfortable asking questions in our Slack channel, so we need a few people ready to answer them.
+
+## Table helpers
+
+Other participants might be more comfortable with in-person assistance, so we have a coach sitting at each table, ready to help.
+
+## Roamers
+
+A couple of people can wander the room, providing ad-hoc assistance where other helpers are already occupied.
+
+## Latecomer helpers
+
+Latecomers might need a bit more help coming up to speed, so it’d help them to have a dedicated helper (perhaps not for the whole session, but at least while they get going).
+
+# What we expect of our coaches:
+
+- You’ve run through the course content beforehand, so you’re familiar with what people will be doing, and can explain what’s going on. While that might sound a bit scary, don’t panic! You don’t need to know everything, just enough to be able to point people in the right direction - you can always ask other coaches for help if you’re not sure about a particular question somebody has.
+- You’ll socialise with the participants during the event - we want them to feel welcome! If you see people standing alone during the breaks, have a chat to them and see how they’re going.
```

Comment:
This is a really important point. We can't forbid coaches from hanging out with other coaches, but remind them that we're here for the attendees who probably feel awkward in a room of strangers.